src/libs/actions/Report.js

```diff
@@ -597,10 +608,12 @@ function updateReportWithNewAction(
     // a chat participant in another application), then the last message text and author needs to be updated as well
     if (newMaxSequenceNumber > initialLastReadSequenceNumber) {
         updatedReportObject.lastMessageTimestamp = reportAction.timestamp;
-        updatedReportObject.lastMessageText = messageText;
         updatedReportObject.lastActorEmail = reportAction.actorEmail;
     }
 
+    // Alwasy cut last message
+    updatedReportObject.lastMessageText = cutLastMessage(messageText);
+
```

Comment:
Why did you move this out of `if block`?

Reply:
There might be a chance that this if block is not executed, so last message is not being cut.

Reply:
Then it means that message is set somewhere else and we should cut the message there not here.

Reply:
I have add cuts to possible `lastMessage`, so this is not necessary. Let's remove this.

Reply:
I see that it is important here. We can remove it. So we can just move the line back to the previous location and add the formatting.

Reply:
Done